src/backend/access/aocs/aocssegfiles.c

```diff
@@ -100,9 +100,11 @@ InsertInitialAOCSFileSegInfo(Relation prel, int32 segno, int32 nvp)
 
 	segrel = heap_open(prel->rd_appendonly->segrelid, RowExclusiveLock);
 
-	InsertFastSequenceEntry(prel->rd_appendonly->segrelid,
-							(int64) segno,
-							0);
+	/*
+	 * Create a new entry in gp_fastsequence or increment
+	 * an existing one (it can exist after transaction rollback).
+	 */
+	GetFastSequences(prel->rd_appendonly->segrelid, (int64) segno, 0, 1);
```

Comment:
I think we have to swap argument values so that `minSequence=1` and `numSequence=0` to avoided side-effects. As result, we won't add an additional increment to the fast sequence during initialization.